website/views.py

```diff
@@ -124,7 +124,10 @@ def dashboard(auth):
 
 @must_be_logged_in
 def home(auth):
-    return {}
+    user = auth.user
+    return {
+        'userId': user._id,
```

Comment:
`userId` is being passed to `home.mako`, but I don't see it being used anywhere. Can it be removed?